src/freva/_user_data.py

```diff
@@ -235,17 +235,21 @@ def delete(self, *paths: os.PathLike, delete_from_fs: bool = False) -> None:
                 if delete_from_fs:
                     file.unlink()
 
-    def index(self, *crawl_dirs: os.PathLike, dtype: str = "fs") -> None:
+    def index(
+        self, *crawl_dirs: os.PathLike, dtype: str = "fs", allow: bool = False
+    ) -> None:
         """Index and add user output data to the databrowser.
 
         This method can be used to update the databrowser for existing user data
 
         Parameters
         ----------
         crawl_dirs:
-            The data path(s) that needs to be crawled
+            The data path(s) that needs to be crawled.
         dtype:
             The data type, currently only files on the file system are supported.
+        allow:
+            Allow indexing to continue on error.
```

Comment:
I think `allow` isn't really descriptive. How about something like `abort_on_errors`, or `continue_on_errors`? Depending how you want to define the default case.

Reply:
`continue-on-errors`